src/serac/physics/thermal_conduction.cpp

```diff
@@ -56,6 +56,8 @@ ThermalConduction::ThermalConduction(int order, const SolverOptions& options, co
   // Default to constant value of 1.0 for density and specific heat capacity
   cp_  = std::make_unique<mfem::ConstantCoefficient>(1.0);
```

Comment:
Is the purpose of these default values to make these parameters essentially optional (since quasi-static problems don't need them)? If so, I suggest putting this in the comment. The current comment repeats what the code already says and doesn't explain intent.

Reply:
Yep. I'll add a comment.